app/controllers/users/enrollment_controller.rb

```diff
@@ -119,13 +119,19 @@ def remove
       return
     end
 
-    remove_assignment_templates
+    remove_assignment_and_enrollment_templates
+
     @course_user.destroy # destroying the course_user also destroys associated Assignments.
 
     render 'users', formats: :json
     update_course_page_and_assignment_talk_templates
   end
 
+  def remove_assignment_and_enrollment_templates
+    remove_assignment_templates
+    make_disenrollment_edits
+  end
```

Comment:
Created this because got an "Method has too many lines" offense from rubocop

Reply:
I think a better way to reorganize this code would be to extract some of the setup and guard code at the start of the method. Ideally, the code in a method is at roughly the same level of abstraction, and the most detailed bits are the first ones to extract.

Extracting these two lines does not really make the method any easier to understand, which is the purpose behind that rubocop rule.

Reply:
Makes sense, I see your point. I tried to do that on 99392335e6128fb183c5a182d25d3c536deb9b7d